graphyte.py

```diff
@@ -28,10 +28,11 @@
 
 class Sender:
     def __init__(self, host, port=2003, prefix=None, timeout=5, interval=None,
-                 queue_size=None, log_sends=False, protocol='tcp'):
+                 queue_size=None, log_sends=False, protocol='tcp', batch_size=1000):
         """Initialize a Sender instance, starting the background thread to
         send messages at given interval (in seconds) if "interval" is not
-        None. Default protocol is TCP; use protocol='udp' for UDP.
+        None. Send at most "batch_size" messages per socket send operation. (default=1000).
```

Comment:
Please remove the extra `.` after "operation" (there's already a `.` after the default parenthesis).